docs/lazy.md

```diff
@@ -148,6 +148,24 @@ $di->setters['Example']['addBars'] = $di->lazyArray([
 ]);
 ```
 
+If you need to modify the LazyArray after you've assigned it we provide two methods (`append($value, $key = null)` and `getArrayCopy()` to help.
+
+```php
+$di->setters['Example']['addFoos'] = $di->lazyArray([]);
+
+// Append to array
+$di->setters['Example']['addFoos']->append($di->lazyNew('SecondFoo'));
+
+// Get a copy of the internal array
+$copy = $di->setters['Example']['addFoos']->getArrayCopy();
```

Comment:
While I agree with the append, I wonder whether manipulation like this is good or not.